cmd/cms.go

```diff
@@ -0,0 +1,189 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path"
+	"regexp"
+	"time"
+)
+
+// TODO:
+// 1. Add support for active-passive directories
+
```

Comment:
look at events command (cmd package) and its clean separation in events package

Reply:
done